includes/Block/Web_Stories_Block.php

```diff
@@ -163,6 +187,18 @@ public function register() {
 						'default' => [],
 					],
 				],
+				'title'           => __( 'Web Stories', 'web-stories' ),
+				'description'     => __( 'Embed Web Stories.', 'web-stories' ),
+				'keywords'        => [
+					/* translators: block keyword. */
+					__( 'embed', 'web-stories' ),
+					/* translators: block keyword. */
+					__( 'web stories', 'web-stories' ),
+					/* translators: block keyword. */
+					__( 'story', 'web-stories' ),
+					/* translators: block keyword. */
+					__( 'stories', 'web-stories' ),
+				],
```

Comment:
Don't think these are needed here. The internationalization should work for PHP, just not in JS.